.evergreen/generated_configs/variants.yml

```diff
@@ -740,8 +740,8 @@ buildvariants:
       PYTHON_BINARY: /Library/Frameworks/Python.Framework/Versions/3.9/bin/python3
   - name: pyopenssl-rhel8-python3.10
     tasks:
-      - name: .replica_set .auth .ssl .sync
-      - name: .7.0 .auth .ssl .sync
+      - name: .replica_set .auth .ssl .sync_async
+      - name: .7.0 .auth .ssl .sync_async
```

Comment:
If we're now running all the async tests when pyopenssl is installed, then I think we can remove the new test_pyopenssl_ignored_in_async test added in test_ssl.

Reply:
hmm, i changed it so that *some* of the pyopenssl variants run async, not all. Should I make them all run async tests and remove that test in test_ssl?